### DISCUSSION TITLE
FreshRSS Logo

### DISCUSSION THREAD

**Discussion by Tealk** [FreshRSS/FreshRSS#4307] [General]
Hello,

is there somewhere a FreshRSS logo without text or some kind of press kit? Would like to use that for a news article.

**Comment by Alkarex** [comment]
Hello,
See the following:

* https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/icons/favicon.svg
* https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/icons/icon.svg
* https://github.com/FreshRSS/FreshRSS/blob/edge/p/themes/icons/FreshRSS-logo.svg

From https://github.com/FreshRSS/FreshRSS/tree/edge/p/themes/icons

You are welcome to share the news article

**Reply by Tealk** [reply]
Thanks for the quick reply and the pictures.

Is not so special, I publish my instance and about each service I inform in a small news.
But I will post a link here :)

**Comment by Tealk** [comment]
Here you can find the article:
https://rollenspiel.monster/news/neuer-dienst-freshrss/
